pkg/services/sqlstore/migrations/correlations_mig.go

```diff
@@ -63,4 +62,8 @@ func addCorrelationsMigrations(mg *Migrator) {
 	mg.AddMigration("add provisioning column", NewAddColumnMigration(correlationsV2, &Column{
 		Name: "provisioned", Type: DB_Bool, Nullable: false, Default: "0",
 	}))
+
+	mg.AddMigration("add type column", NewAddColumnMigration(correlationsV2, &Column{
```

Comment:
Wouldn't it make sense to make this `nullable: false` and migrate now with default value of `query`? That way we prevent any accidental `null` values in the future and we don't have to do anything special on read path.

Reply:
So I don't think this kind of migration should be anything special, we already have the same non nullable with default columns in other places like here https://github.com/grafana/grafana/blob/d8904f3ca46cc8058036e5d01eba39f759fd581a/pkg/services/sqlstore/migrations/datasource_mig.go#L119

So yeah I can see it is failing but there seems to be something else going on which I think we should investigate a bit.

Reply:
```
migrations_mock.go:43: failed to initialize sql store: migration failed (id = add type column): Error 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'query' at line 1
```

The addColumn SQL statement that the migrator code creates does not seem to be very dialect specific so I wonder if the problem is `query` string itself, maybe it is a reserved word that needs to be quoted in some special way.

Reply:
Would helpful to enable debug logging there for this as this log line https://github.com/grafana/grafana/blob/5dfe4cf407d0ae27249435693d97ac2710262267/pkg/services/sqlstore/migrator/migrator.go#L280 would be helpful.

Reply:
This was fixed. A lot easier than I was thinking, just needed to look at other similar migrations that have text.